src/modules/comments.js

```diff
@@ -0,0 +1,81 @@
+export const counter = (comment) => {
+  let commentCounter = comment.length;
+  if (comment.error) {
+    commentCounter = 0;
+  }
+  return commentCounter;
+};
+
+export const showComment = (data, commentInfo) => {
+  commentInfo.innerHTML = '';
+  const commentTitle = document.createElement('h4');
+  commentTitle.textContent = `Comments (${counter(data)})`;
+  commentInfo.appendChild(commentTitle);
+  const commentItem = document.createElement('div');
+  commentItem.classList.add('comment-items');
+  if (!data.error) {
+    // eslint-disable-next-line
+    for (const element of data) {
```

Comment:
- [ ] Same here, Kindly remove the eslint disablement line and try to solve all eslint errors yourself.